### PR TITLE
Two Volcanoes Map by Oliver4

### PR DESCRIPTION
**Pull request by AnyBlue**
New map by Oliver4

### CODE REVIEW

**Comment by olliy1or** [comment]
The map has no license

**Comment by AnyBlue** [comment]
> The map has no license

I will ask him.

File: volcanoes/map.conf [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+team.2.pos = (15,-63,-97)
+team.1.pos = (16,-63,101)
+team.1.color = red
+team.1 = red
+author = Oliver4
+rotation = z
+h = 140
+team.2 = blue
+name = Two Volcanoes
+initial_stuff = default:pick_steel,default:torch 10,shooter_guns:pistol
```
Comment (Avyukt1) [inline volcanoes/map.conf:11]: This gives an unloaded pistol,
```suggestion
initial_stuff = default:pick_steel,default:torch 10,shooter_guns:pistol_loaded
```

**Review by Avyukt1 — CHANGES_REQUESTED** [review]

**Comment by Avyukt1** [comment]
This map doesn't have bugs. Also It has a good treasure. But the land flat and I think you should add more trees or mountains in the free space.

**Comment by olliy1or** [comment]
imo the map is a little plain 
cant use the volcanoes for anything much
maybe add a little more builds to the sides and near the bases?

**Comment by JostP** [comment]
Map works perfectly, but unfortunately it map won't be added:
- very plain, it isn't unique , we already have to many of them.
- we already have to many maps with good ores such as mese and diamond.

**Pull request closed without merging**